main/main.go

```diff
@@ -387,17 +387,32 @@ func replicateAction(c *cli.Context) error {
 	var tablelist []string
 	var whitelist bool
 	var regex bool
-	if len(wl) != 0 && len(bl) != 0 && len(wlr) != 0 && len(blr) != 0 {
-		return errors.New("'whitelist', 'whitelist-regex', 'blacklist', and 'blacklist-regex' are mutually exclusive. Use one or the other but not both")
-	} else if len(wl) != 0 {
+	filterConfigCount := 0
+	if len(wl) != 0 {
 		whitelist = true
 		tablelist = wl
-	} else {
+		regex = false
+		filterConfigCount++
+	} else if len(wlr) != 0 {
+		whitelist = true
+		tablelist = wlr
+		regex = true
+		filterConfigCount++
+	} else if len(bl) != 0 {
 		whitelist = false
 		tablelist = bl
+		regex = false
+		filterConfigCount++
+	} else if len(blr) != 0 {
+		whitelist = false
+		tablelist = blr
+		regex = true
+		filterConfigCount++
 	}
 
-	regex = (len(wlr) != 0 || len(blr) != 0)
+	if filterConfigCount > 1 {
```

Comment:
This check should go above the if else chain because `filterConfigCount` will never go above 1.

Reply:
I see what you mean.  A better fix might be to change the if-else chain into just if-statements so that `filterConfigCount` actually adds up instead of defaulting to whatever the first match is in case of multiple, conflicting configurations.